index.html

```diff
@@ -10711,13 +10711,22 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
-				<p>In some user agents, multiple reference relationships for descriptive information are not supported by the accessibility API. In such cases, if both <pref>aria-describedby</pref> and <code>aria-details</code> are provided on an element, <code>aria-details</code> takes precedence.</p>
+				<p>Identifies the <a>element</a> (or elements) that provide additional, extended information about the <a>object</a>. See related <pref>aria-describedby</pref>.</p>
+				<p>The additional information can be:</p>
+				<ul>
+					<li>A description or general annotation (this is the default).</li>
+					<li>A <rref>comment</rref> &mdash; in this case the element referred to SHOULD use the <rref>comment</rref> role.</li>
+					<li>A <rref>definition</rref> &mdash; in this case the referring element SHOULD use the <rref>term</rref> role and the element referred to SHOULD use the <rref>definition</rref> role.</li>
+					<li>A footnote &mdash; in this case the element referred to SHOULD use the <code>doc-footnote</code> role from [[DPUB-ARIA]].</li>
+					<li>An endnote &mdash; in this case the element referred to SHOULD use the <code>doc-endnote</code> role from [[DPUB-ARIA]].</li>
+				</ul>
+
+				<p>The <code>aria-details</code> attribute references elements that provide more detailed information than would normally be provided by <pref>aria-describedby</pref>. It enables <a>assistive technologies</a> to make users aware of the availability of extended information as well as navigate to it. Authors SHOULD ensure elements referenced by <code>aria-details</code> are visible to all users.</p>
+				<p>Unlike elements referenced by <code>aria-describedby</code>, elements referenced by <code>aria-details</code> are not used in the Accessible <a href="https://www.w3.org/TR/accname-aam-1.1/#mapping_additional_nd_description">Description Computation</a> as defined in the Accessible Name and Description specification [[ACCNAME-1.1]]. Thus, the content of elements referenced by <code>aria-details</code> are not flattened to a string when presented to assistive technology users. This makes <code>aria-details</code> particularly useful when converting the information to a string would cause a loss of information or make the extended information more difficult to understand.</p>
+				<p>It is valid for an element to contain both <code>aria-details</code> and a description via <code>aria-describedby</code> or <code>aria-description</code>. In this case, user agents should minimally expose the <code>aria-details</code> relation and the computed flat description string. In user agents that cannot expose multiple descriptive relations, the first <code>aria-details</code> takes precedence.</p>
```

Comment:
This paragraph is implying normative UA requirements and offering new information about aria-details as well as making comparisons with other properties. I think the present wording could be hard to follow for some who are not intimately familiar with the differences among aria-details, aria-describedby, and aria-details. I also think the UA requirements should be more explicit. I suggest the following.

```suggestion
				<p>The <code>aria-details</code> property supports referring to multiple elements. For example, a paragraph in a document editor may reference multiple comments that are not related to each other. If a user agent relies on an accessibility API that does not support exposing multiple descriptive relations, the user agent SHOULD expose the relationship to the first element referenced by <code>aria-details</code>.</p>
				<p>It is valid for an element to have both <code>aria-details</code> and a description specified with either <code>aria-describedby</code> or <code>aria-description</code>. If a user agent relies on an accessibility API that does not support exposing multiple descriptive relations, and if an element has both <code>aria-details</code> and <rref>aria-describedby</rref>, the user agent SHOULD expose the <code>aria-details</code> relation and the description string computed from the <rref>aria-describedby</rref> relationship.</p>
```